Automattic/jetpack
# Add component for authenticated experiments to ExPlat

See related issue

## Proposed changes:

* This PR implement another set of hooks and components to be used specifically for running experiments for authenticated users.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


Yes. It adds a way to run A/B experiments specifically with authenticated users.

## Testing instructions:






* Confirm the code looks good and it builds correctly
* Follow the test instructions in https://github.com/Automattic/jetpack/pull/37973
